Riot/Modules/Room/RoomCoordinator.swift

```diff
@@ -32,6 +32,7 @@ final class RoomCoordinator: NSObject, RoomCoordinatorProtocol {
     private let userIndicatorStore: UserIndicatorStore
     private var selectedEventId: String?
     private var loadingCancel: UserIndicatorCancel?
+    private var secondaryIndicatorCancel: UserIndicatorCancel? // Used for low priority advertizements
```

Comment:
I wonder if long term it might make sense to have user indicators stored per service, e.g. have this as a `locationSharingIndicator`? If in the future a `superDuperService` called `startSecondaryLoading(withMessage: "Magical things are happening")` whilst the location sharing service was presenting an indicator, then it that indicator will simply be thrown away as it will hit the `guard secondaryIndicatorCancel == nil`.

Reply:
Yes I can use a specific one. In fact it will disappear once the spinner will be in the banner directly.